src/components/DisplayedDocument.ts

```diff
@@ -275,7 +273,6 @@ function DisplayedDocument(): m.Component<Attrs> {
      * will be longer than our max node width
      *
      * @param contents The contents of the node to be rendered
-     *
```

Comment:
Another thing I'm curious about, as this didn't trigger a linting error for me.

Reply:
Looks ilke this was another auto-fix - came from `'plugin:jsdoc/recommended'` (`jsdoc/tag-lines` rule). Undoing the fix, appears it only shows as a warning in VSCode (though `--fix` fixes it automatically):

<img width="186" alt="Screen Shot 2022-02-03 at 8 33 16 PM" src="https://user-images.githubusercontent.com/2965457/152472288-b54d9ecb-c1c1-495a-9377-f940395178a0.png">

Reply:
Also visible now that my eslint is fixed :stuck_out_tongue: .